### PR TITLE
Edit comments and add implementations for the histogram function

### LINKED ISSUE #17: Code for Histogram

### LINKED ISSUE #22: Test script for Histogram

### PR DESCRIPTION
**Pull request by shyan0903**
Please take a look at my code and feel free bring up any questions or concerns to help me improve! Thank you! I will continue to work on the test functions.

### CODE REVIEW

**Comment by ruben1dlg** [comment]
Looks very good! I just have two comments on the code:
- I would probably add a comment when you call the 'validate' function within the 'histogram' function, just mentioning that it is defined below. It threw me off a little bit the first time I read it 😀.
- I think it would be a good idea to add a parameter to the function that allows the user to change the number of bins. This could be useful specially when you are using a numeric variable on the x-axis. I tried the code and it works properly! That's the only quick upgrade I can think about

File: src/magmaviz/histogram.py [hunk 1]
```diff
@@ -20,3 +27,63 @@ def histogram(df, x, y):
...
+    validate(df, y)
+  
+    chart = alt.Chart(df).mark_bar().encode(
+        x=x,
+        y=y,
+        color=alt.Color(y, scale=alt.Scale(scheme="magma"))
+    )
+    return chart
+
+def validate(df, y):
```
Comment (iamMoid) [inline src/magmaviz/histogram.py:53]: Please add the docstring to explain what the function is doing.

**Review by iamMoid — CHANGES_REQUESTED** [review]
Looks good! A docstring for the validate function and this should be good for approval.

**Comment by shyan0903** [comment]
Thank you all for your reviews! I have incorporated the necessary changes.

**Review by iamMoid — APPROVED** [review]
Looks good!

**Review by ruben1dlg — APPROVED** [review]

**Pull request merged**